main.cpp

```diff
@@ -214,9 +262,14 @@ struct ControlRoom
     bool switchStudioPower(); // returns state of studio power
 };
 
+ControlRoom::ControlRoom()
+{
+    std::cout << "ControlRoom being constructed." << std::endl;
+}
+
 void ControlRoom::seatEngineer(std::string engineerName)
 {
-        std::cout << "Today we're enjoying the mixing skills of  " << engineerName;      
+        std::cout << "Today we're enjoying the mixing skills of " << engineerName << std::endl;      
```

Comment:
Fix this indentation.
Code should be indented 4 spaces from the surrounding curly braces
- [ ] revise

Reply:
revised